### DISCUSSION TITLE
NEED MEMBERS FOR MY NEW ORGANIZATION TO COLLEBRATE AI MODELS

### DISCUSSION THREAD

**Discussion by kir-hakeem** [community/community#133106] [Programming Help]
### Select Topic Area

Question

### Body

If someone is interested to collaborate for AI models please say hi!! I will add you to organization

**Comment by SamuelOUS** [comment]
Hi kir-hakeem! Add me :)

**Comment by ebndev** [comment]
Hi @kir-hakeem, thanks for sharing with the community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
